infrastructure/src/main/java/org/corfudb/infrastructure/log/FileSystemAgent.java

```diff
@@ -43,6 +50,9 @@ public final class FileSystemAgent {
 
     private final PartitionAttribute partitionAttribute;
 
+    private static final ScheduledExecutorService scheduler =
```

Comment:
Aren't we missing the shutdown for this new thread executor? Or am I missing something?

Reply:
fixed